src/App.js

```diff
@@ -1,26 +1,63 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
 
-function App() {
+import Board from './components/Board';
+
+const PLAYER_1 = 'X';
+const PLAYER_2 = 'O';
+
+const generateSquares = () => {
+  const squares = [];
+
+  let currentId = 0;
+
+  for (let row = 0; row < 3; row += 1) {
+    squares.push([]);
+    for (let col = 0; col < 3; col += 1) {
+      squares[row].push({
+        id: currentId,
+        value: '',
+      });
+      currentId += 1;
+    }
+  }
+
+  return squares;
```

Comment:
Adding this comment here from the other PR since it feels more relevant here:

Do we want to update the starter code to just create an array of 9 squares since we are relying on styling to actually dictate the number of squares per row anyway?
It just seems a little funky to me to give them an array of arrays that they immediately flatten and do nothing meaningful with the nesting of the original array.

Reply:
I kept it a 2D array because it's much easier to conceptually figure out a winner this way.  Maybe I'm wrong and feel free to overrule me.